### ISSUE TITLE
Convert Translate command

### ISSUE THREAD

**Issue by andrewtavis** [scribe-org/Scribe-iOS#41]
This issue is of great importance to Scribe, but is currently blocked, and as of now isn't a priority while initial improvements are taking place.

This issue would focus on the eventual translation feature where newly added translation lexeme data would be linked to the translation selection UI. Closing this issue would thus close the project, end the **`beta`** status of `Translate`, lead to an app update, and with that countless other issues to update the media surrounding Scribe to show off this new and exciting feature.

Feedback on how best to achieve this would be much appreciated!

- This issue is blocked by scribe-org/Scribe-iOS#49 and scribe-org/Scribe-Data#17
- The ideal version of this issue is blocked by scribe-org/Scribe-Data#18 and scribe-org/Scribe-iOS#20 as the overarching goal for `Translate` is having synonyms for educated translation selections

**Comment by andrewtavis** [comment]
Deleting as the issues that are included in here don't need an issue to organize them :)

**Comment by andrewtavis** [comment]
Reopened as this would be the issue where the translation feature is changed over from Hugging Face :)

**Comment by andrewtavis** [comment]
Closing again as the first part of the work has been finished in #476. We'll make a new issue when the time comes :)
